spring-cloud-dataflow-docs/src/main/asciidoc/streams.adoc

```diff
@@ -131,6 +131,13 @@ configuration.classes=org.springframework.cloud.stream.app.file.sink.FileSinkPro
 configuration-properties.names=server.port
 ```
 
+[IMPORTANT]
+====
+As of Spring Cloud Data Flow `1.0.0.RELEASE` the whitelisting of application properties
+is only supported for Spring Boot `1.3.x` based application. Milestone releases
+of the upcoming `1.4.0` release are not supported, yet.
+====
```

Comment:
I think we could also add what happens when someone uses `@EnableDataflowServer` that uses boot `1.4.0` with the apps using `1.3.x` or `1.4.x` boot versions.